extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/JsonWebKeySet.java

```diff
@@ -37,6 +38,9 @@ private void initKeys(String json) {
             org.jose4j.jwk.JsonWebKeySet jwkSet = new org.jose4j.jwk.JsonWebKeySet(json);
             for (JsonWebKey jwkKey : jwkSet.getJsonWebKeys()) {
                 if (isSupportedJwkKey(jwkKey)) {
+                    List<Key> keysForAlgorithm = keys.computeIfAbsent(jwkKey.getKeyType(), k -> new ArrayList<>());
```

Comment:
This is fine but there is a general preference to avoid lambda expressions of all kind. Can you please consider pushing the code used to handle such a Map just below into a private function that can be reused for collecting all keys as well ?

Reply:
removed lambda usage and added a reusable method for the "list in a map" case.